### PR TITLE
Syringe guns new sprite and become bulky

### PR DESCRIPTION
**Pull request by UnokiAs**
## About The Pull Request

This PR change the syringe gun sprite, the syringe gun is  now bulky and can only be placed on back slot or suit storage with labcoat, medical windercoat and biosuit.

https://user-images.githubusercontent.com/82665345/139691016-7bc5236c-125f-4f45-9b88-1c66db1f2394.mp4

Replaced all the roundstart spawning old syringe guns with  new syringe guns.
~~Reduce the price in tc of syringe dart gun (4 to 2).~~

## Why It's Good For The Game

This change is aimed to reduce the easy use of death chems by the crew by making the syringe gun less likely to be hoarded by paramedics, etc... and also reducing the surprise factor.

This pr doesn't really affect antags in general since you can still use the syringe gun on suit slot (labcoat, wintercoat, biosuit) and shoot as fast.

This doesn't really affect the hulk meta since most player don't use mutadone to counter them and instead rely on lethals anyways.

~~Also make the syringe dart gun have a appeal to be used.~~

## Changelog

:cl:Unoki, Kryson
balance: Regular medical syringe guns have been replaced with syringe rifles
imageadd:  new dna/dart/medical Syringe gun sprite
/:cl:

### CODE REVIEW

**Comment by UnokiAs** [comment]
>What about rapid syringe gun?

I was thinking about it but I rather wait for this pr to get merge avoid making sprite for nothing lel, Also another reason rapid syringe gun are not really that common in game

**Comment by TheBonded** [comment]
Does this change how the syringe gun acts, or is it just new sprites + bigger size? It would be cool if it allowed it to hold 3 shots in it, but would need to pull back the bolt, similar to the mosin. It would also be neat to see the syringe fly faster perhaps, or have it possible to store bluespace syringes in the gun. Another interesting change that could be made would be to give the syringe rifle an internal chem storage, maybe 50-100 chems, so you could slide empty syringes in and they would take from the internal supply of chems.

These are fairly underdeveloped ideas, I was just thinking of ways to make this change make the weapon act more like a rifle, increasing utility while still keeping it worse for what it's currently used for

File: code/modules/projectiles/guns/special/syringe_gun.dm
Comment (tralezab) [inline code/modules/projectiles/guns/special/syringe_gun.dm:100]: please put at least one space between the last type declaration and the next

**Review by tralezab — CHANGES_REQUESTED** [review]

**Comment by Mothblocks** [comment]
:-1: just make the regular syringe gun bulky, there's no flavor in a "syringe rifle" why would a medical team have that

File: code/modules/projectiles/guns/special/syringe_gun.dm
Comment (UnokiAs) [reply code/modules/projectiles/guns/special/syringe_gun.dm]: Done 👍

**Comment by UnokiAs** [comment]
>Does this change how the syringe gun acts, or is it just new sprites + bigger size? It would be cool if it allowed it to hold 3 shots in it, but would need to pull back the bolt, similar to the mosin. It would also be neat to see the syringe fly faster perhaps, or have it possible to store bluespace syringes in the gun. Another interesting change that could be made would be to give the syringe rifle an internal chem storage, maybe 50-100 chems, so you could slide empty syringes in and they would take from the internal supply of chems.

>These are fairly underdeveloped ideas, I was just thinking of ways to make this change make the weapon act more like a rifle, increasing utility while still keeping it worse for what it's currently used for

I was thinking about putting a bolt action thingy but ye I was thinking if I did that it would be too much maybe, I'm wrong

**Comment by UnokiAs** [comment]
>👎 just make the regular syringe gun bulky, there's no flavor in a "syringe rifle" why would a medical team have that

Eh I could have but I wanted to make and sprite and also I was thinking about the syringe dna gun would have been weird if it wasn't bulky while having the same sprite has the syringe gun

**Comment by Zytolg** [comment]
I'm a fan of the sprite. Looks better than what we currently have.

**Comment by itseasytosee** [comment]
> 👎 just make the regular syringe gun bulky, there's no flavor in a "syringe rifle" why would a medical team have that

Image of a dart rifle as reference
image
While would a medical team have a normal syringe gun anyways?

**Comment by KittyNoodle** [comment]
removing departmental tools that antags can use and not look turbovalid detracts from paranoia. getting hit by a syringe gun dart does not make you think you need to drink multiver now, but getting hit by a micro dart gun dart or a reagent dartgun dart does.

**Comment by itseasytosee** [comment]
> removing departmental tools that antags can use and not look turbovalid detracts from paranoia. getting hit by a syringe gun dart does not make you think you need to drink multiver now, but getting hit by a micro dart gun dart or a reagent dartgun dart does.

I think if you got shot full of unknown chemicals from ANY source you would be concerned.

**Comment by Rohesie** [comment]
Is there a reason you're keeping the syringe gun, instead of replacing it entirely with the rifle here?

If you do that I might merge it, as long as the spritework is approved. I like the idea.

**Comment by antropod** [comment]
another PR that nerfs syringe guns?

**Comment by antropod** [comment]
> Does this change how the syringe gun acts, or is it just new sprites + bigger size? It would be cool if it allowed it to hold 3 shots in it, but would need to pull back the bolt, similar to the mosin. It would also be neat to see the syringe fly faster perhaps, or have it possible to store bluespace syringes in the gun. Another interesting change that could be made would be to give the syringe rifle an internal chem storage, maybe 50-100 chems, so you could slide empty syringes in and they would take from the internal supply of chems.
> 
> These are fairly underdeveloped ideas, I was just thinking of ways to make this change make the weapon act more like a rifle, increasing utility while still keeping it worse for what it's currently used for

Ability to hold bluespace syringes was removed from syringe gun.

Item with internal chem storage is called "Reagent Dartgun" and it is 14 TC uplink item.

3 shots is half of rapid syringe gun.

**Comment by itseasytosee** [comment]
@Rohesie  One advantage I see to keeping the default syringe gun rather than replacing it is all of the other syringe gun subtypes do not need to be updated to be small and use separate in-hands. The normal syringe gun serves as a blueprint for all of the subtypes despite not appearing in game.

**Comment by Spookuni** [comment]
I'm fine with switching this out to bulky / suit storage, quickdrawing a syringe gun out of a bag is a silly easy way to go from non-threatening to instant death in no time flat, but if the size is increased it should be suit storage equipable by doctors, paramedics and chemists.

The syringe gun IS a piece of medical kit, medical's primary method of defending themselves from incursion and also has some really neat niche uses for on the go ranged healing (Shooting dying people with 5u atropine 5u salicylic and 5u oxandralone syringes to get them off the floor at range is peak paramedic gameplay I will not be dissuaded from this). Changing it into a security focused piece of kit (either carry it around in your hands or the time or wear armour, I.E Be security or raid security) feels like a massive drop in medical's ability to use their roundstart kit to interact with the round, both for funny healing dart uses and defensive action - making the syringe gun need to be open carried in medical suit storages already accomplishes the goal of making it harder to use offensively - since you can't quickdraw it out of bags on people.

The syringe gun isn't sec gear, and shouldn't be sec gear, they've already got shotgun darts if they want to get chemsy, the syringe gun shouldn't be relegated to their use or shitty use.

**Comment by Coastfront** [comment]
With this change i feel like doctors/the cmo/ chemists/ fucking anyone in med should be able to suit storage it without needing to break into sec to steal body armor. Since why does an item that security has no reason to use (they already get chem shells which iirc are better in every way over syringes, that is loaded into a 7 shot gun that they have like 5 of roundstart.)
As others have said syringe gun is the only real thing med has to defend themselves from t h e t i d e and making it something that they either need to beg security for, raid their department sec outpost locker, or just hand over to security because they can atleast stow the damned thing is just kinda sucky.

**Comment by optimumtact** [comment]
I think the original intent of the syringe gun was a multi shot injector needle for close range use.

The dart rifle makes much more sense for what the syringe gun eventually morphed into.

**Comment by Bawhoppen** [comment]
I am still wary of the sprite here even after the revisions we discussed in spriterbus. It still looks pretty funky overall, and that back syringe plunger hasn't been shrunk down yet either (well removed rather since the syringe slot is so far forward now). Of course I'll leave that up to Kryson and Twaticus to opine in on.

No opinion on balance.

**Comment by ArcaneDefence** [comment]
> I think the original intent of the syringe gun was a multi shot injector needle for close range use.

Isn't this the hypo now?

**Comment by Mothblocks** [comment]
> Eh I could have but I wanted to make and sprite and also I was thinking about the syringe dna gun would have been weird if it wasn't bulky while having the same sprite has the syringe gun, Tho if the pr get close because of the sprite, I will remake this pr whitout it

You can keep the sprite just name it "syringe gun" because "syringe rifle" isn't even pretending, then name the others mini-syringe gun or whatever

**Comment by Mothblocks** [comment]
> another PR that secretly nerfs syringe guns?

"Secretly"?

**Comment by UnokiAs** [comment]
>Is there a reason you're keeping the syringe gun, instead of replacing it entirely with the rifle here?
>
>If you do that I might merge it, as long as the spritework is approved. I like the idea.

I kept the /gun/syringe to avoid to have to make a weight class for every subtype and also for the dna gun, Another issue would be  some don't have a backsprite like the blowgun.
 
>The syringe gun IS a piece of medical kit, medical's primary method of defending themselves from incursion and also has >some really neat niche uses for on the go ranged healing (Shooting dying people with 5u atropine 5u salicylic and 5u >oxandralone syringes to get them off the floor at range is peak paramedic gameplay I will not be dissuaded from this). >Changing it into a security focused piece of kit (either carry it around in your hands or the time or wear armour, I.E Be >security or raid security) feels like a massive drop in medical's ability to use their roundstart kit to interact with the round, >both for funny healing dart uses and defensive action - making the syringe gun need to be open carried in medical suit >storages already accomplishes the goal of making it harder to use offensively - since you can't quickdraw it out of bags on >people.

I will maybe make it able to be wear by the labcoat subtype, paramedic vest and winter coat.

>You can keep the sprite just name it "syringe gun" because "syringe rifle" isn't even pretending, then name the others >mini-syringe gun or whatever

Alright I will rename the syringe rifle into syringe gun and the other mini syringe gun

**Comment by UnokiAs** [comment]
-Syringe rifle are nenamed back  to syringe gun
-Syringe gun can be equiped on suit storage of labcoat, Wintercoat Medbay.

**Comment by UnokiAs** [comment]
>I am still wary of the sprite here even after the revisions we discussed in spriterbus. It still looks pretty funky overall, and that >back syringe plunger hasn't been shrunk down yet either (well removed rather since the syringe slot is so far forward now). 
>Of course I'll leave that up to Kryson and Twaticus to opine in on.

Capture d’écran 2021-10-30 175346

This look better ?

**Comment by Rohesie** [comment]
If you don't want to wait until the spritetainers review it, you can make the balance change first, making the base syringe gun bulky and held only by backpacks and doctor suits, and then make the sprite and flavor change in a different PR.

**Comment by anclandor** [comment]
This makes syringe guns objectively worse than shotguns.  Previously it was a trade off, stealth versus having a larger clip and 30u AP syringes.  At the very least, make an option for sawing off the barrel of syringe guns.

**Comment by Iplayforss13** [comment]
Saw of barrel, but less accuracy, seems reasonable, just like that 1 bulky mosin pr

**Comment by Iplayforss13** [comment]
Also with the bigger catridge, Im thinking bluespace syringes can be fit in

**Comment by UnokiAs** [comment]
> This makes syringe guns objectively worse than shotguns.  Previously it was a trade off, stealth versus having a larger clip and 30u AP syringes.  At the very least, make an option for sawing off the barrel of syringe guns.  
> 

Why would a syringe gun every doc has access to, be more useful than sec equipment? And also It's VERY rare to see someone use darts shells right now everyone use improvised shells.

**Comment by UnokiAs** [comment]
> Saw of barrel, but less accuracy, seems reasonable, just like that 1 bulky mosin pr

Could be something, Maybe

**Comment by UnokiAs** [comment]
Putting this pr on draft still working on sprites.

**Comment by anclandor** [comment]
> > This makes syringe guns objectively worse than shotguns.  Previously it was a trade off, stealth versus having a larger clip and 30u AP syringes.  At the very least, make an option for sawing off the barrel of syringe guns.
> 
> Why would a syringe gun every doc has access to, be more useful than sec equipment? And also It's VERY rare to see someone use darts shells right now everyone use improvised shells.

Never mind, I was thinking that this was also a nerf for the traitor syringe gun, and in that case traitors would just break into armory, but if it is just for normal syringe gun I guess it is fine.

**Comment by UnokiAs** [comment]
> > > This makes syringe guns objectively worse than shotguns.  Previously it was a trade off, stealth versus having a larger clip and 30u AP syringes.  At the very least, make an option for sawing off the barrel of syringe guns.
> > 
> > Why would a syringe gun every doc has access to, be more useful than sec equipment? And also It's VERY rare to see someone use darts shells right now everyone use improvised shells.
> 
> Never mind, I was thinking that this was also a nerf for the traitor syringe gun, and in that case traitors would just break into armory, but if it is just for normal syringe gun I guess it is fine.  

I'm also planning to reduce the tc cost of the dart gun after this pr.

**Comment by Iplayforss13** [comment]
Why not just change the tc price in the same pr?

**Comment by Coastfront** [comment]
> Why not just change the tc price in the same pr?

because then maintainers spam "atomize" 3,000 times in the comments

**Comment by Krysonism** [comment]
bild

I drew up some possible syringe rifle variations in the spriting channel.

I made them 40px wide in order to avoid the stubby look of other rifles. I know other codebases have rifles that are up to 48px wide, so hopefully this should work despite some spillover.

**Comment by Istoprocent** [comment]
There is next to no reason to change current syringe gun, since it works as intended. In a game with instant ggs such as flashes (available to all without there being not enough sunglasses available to the crew) the syringe gun pales in comparison. Could there be an improved and printable version of syringe gun - sure, let it shoot 30u or 45u syringes or take a clip of syringes and have a reload mechanism comparable to mosin (double z, 5 rounds).

The issue might be the mixes rather than the gun and most of them can easily be avoided by getting a free 10u multiver pill from the chemistry smart fridge.

The paramedics taking the syringe gun might be the best use case for the gun, since a) there is enough people with circular saws in the medbay to subdue hostile individuals and b) they usually don't opt for chemistry mixes such as gunpowder.

**Comment by Mothblocks** [comment]
> In a game with instant ggs such as flashes (available to all without there being enough sunglasses available to the crew) the syringe gun pales in comparison. 

This is called power creep

**Comment by UnokiAs** [comment]
-New sprite thanks to Kryson
-Biosuit can hold the syringe gun now

Capture d’écran 2021-11-01 135551

**Comment by UnokiAs** [comment]
When you have a conflict but no idea what make the conflict aaaa

**Review by Krysonism — APPROVED** [review]
The sprites look good.

**Comment by KittyNoodle** [comment]
Traitors are not he only antag and in general balancing the game around having to use tc instead of using things departmentally available to you is bad

**Comment by UnokiAs** [comment]
All done well the conflict in bio.dm is dumb because I need to add /obj/item/gun/syringe/medical :/

**Comment by jughu** [comment]
ok tell me whats stopping me from making a improvised shotgun with chem darts instead which is functionally the same gun

**Comment by Coastfront** [comment]
> ok tell me whats stopping me from making a improvised shotgun with chem darts instead which is functionally the same gun

The fact that the improv shotgun got removed from the game

**Comment by UnokiAs** [comment]
>ok tell me whats stopping me from making a improvised shotgun with chem darts instead which is functionally the same gun

More effort than syringe gun so great 👍

**Comment by jughu** [comment]
> > ok tell me whats stopping me from making a improvised shotgun with chem darts instead which is functionally the same gun
> 
> The fact that the improv shotgun got removed from the game

bartender shotgun 🥺

**Comment by Coastfront** [comment]
> ok tell me whats stopping me from making a improvised shotgun with chem darts instead which is functionally the same gun

Okay other than me shitposting about that the improv shotgun is now the pipegun, List of downsides on the pipegun loaded with chem shells vs the new syringe gun.

Maint rifle,Has a slightly longer reload, a more annoying to produce ammo type (chem shells only available at cargo, while people who would have acess to the syringe guns could just mass print syringes.)
Maint rifle also, can jam. With its jam chances increasing with each shot, while the syringe gun, does not.
Syringe gun is roundstart available, maint rifle, is not.
Maint rifle also gets your teeth kicked in by sec if they dont like you having it, while most people wont give a shit if chemistry/medical has acess to the item that spawns in their department at the start of the round every round.
Chemical shotgun shells cannot pierce eva gear, the syringe rifle can chamber AP syringes for injecting whomever you please with your heparin.

In total, the maint rifle is less convenient for medical staff to use and make ammo for, is more suspicious, And can literally jam, getting you killed by not being able to shoot.

File: code/modules/projectiles/guns/special/syringe_gun.dm
Comment (Rohesie) [inline code/modules/projectiles/guns/special/syringe_gun.dm:101]:
```suggestion
	force = 6
```
This comment pertains to the PR, but not to the code.
People reading it in the future won't know what it is about.

**Comment by Istoprocent** [comment]
The sprites look cool and all, but there is no reason for this change (other than "oh shit cool sprites" and I feel that this is the only reason for this PR). Introduce a new weapon that has a scope or does something extraordinary such as having internal chem reservoir (like sec has bulky ebows, this could be bulky reagent gun or whatever) and add it to the sci research as medium-late game medical thing - cargo has guns, every medical could have AP syringe rifles.

Getting access to cargo and pipe rifle is easier than getting access to syringe gun and syringes (unless you print syringes at cargo, then you get those really easily).

**Comment by Coastfront** [comment]
> The sprites look cool and all, but there is no reason for this change (other than "oh shit cool sprites" and I feel that this is the only reason for this PR). Introduce a new weapon that has a scope or does something extraordinary such as having internal chem reservoir (like sec has bulky ebows, this could be bulky reagent gun or whatever) and add it to the sci research as medium-late game medical thing - cargo has guns, every medical could have AP syringe rifles.
> 
> Getting access to cargo and pipe rifle is easier than getting access to syringe gun and syringes (unless you print syringes at cargo, then you get those really easily).

Syringe gun being harder to make than improv weapons has literally always been a thing. A roundstart medical item being ltierally the exact same as a traitor item is wack.

**Comment by UnokiAs** [comment]
Gaming

**Comment by UnokiAs** [comment]
Made the color darker to match the other sprite
Capture d’écran 2021-11-01 203312

**Comment by Istoprocent** [comment]
> 
> Syringe gun being harder to make than improv weapons has literally always been a thing. A roundstart medical item being ltierally the exact same as a traitor item is wack.

There is a difference between normal syringe gun and traitor syringe gun - the traitor syringe gun **fits in pocket**, was near silent (iirc) and didn't have a text when somebody fired it (iirc).

**Comment by Coastfront** [comment]
> > Syringe gun being harder to make than improv weapons has literally always been a thing. A roundstart medical item being ltierally the exact same as a traitor item is wack.
> 
> There is a difference between normal syringe gun and traitor syringe gun - the traitor syringe gun **fits in pocket**, was near silent (iirc) and didn't have a text when somebody fired it (iirc).

No the dartgun was literally just a smaller syringe gun. thats it. nothing else special about it

**Comment by itseasytosee** [comment]
> No the dartgun was literally just a smaller syringe gun. thats it. nothing else special about it
@Coastfront
You, my sir, are objectively incorrect. It makes much less noise and has no firing text because it has `suppressed = TRUE`
Also, the only thing this pr changes is round start syringe guns. All other varients work exactly as before.

**Comment by Coastfront** [comment]
> > No the dartgun was literally just a smaller syringe gun. thats it. nothing else special about it
> 
> @Coastfront
> 
> You, my sir, are objectively incorrect. It makes much less noise and has no firing text because it has `suppressed = TRUE` Also, the only thing this pr changes is round start syringe guns. All other varients work exactly as before.

I was wrong about the dartgun but i also never said the pr was changing the dartgun, in anyway.

File: code/modules/projectiles/guns/special/syringe_gun.dm
Comment (Rohesie) [inline code/modules/projectiles/guns/special/syringe_gun.dm:188]:
```suggestion
	base_pixel_x = 0
	pixel_x = 0
	force = 4

```
Or do you want to buff it as well?

**Comment by Istoprocent** [comment]
> 
> I was wrong about the dartgun but i also never said the pr was changing the dartgun, in anyway.

The PR is changing dart gun from 4TC to 2TC from what I understand and I have no problem with that (it might have been retroactively removed from the PR).
You did say:"_A roundstart medical item being ltierally the exact same as a traitor item is wack._", suggesting that the current syringe gun is essentially the same as traitor dart gun, which is not the case.

The PR would be better if nu-syringe gun would be at least BELT compatible like other guns such as Laser Rifle/Cargo Guns and so on.

File: code/modules/projectiles/guns/special/syringe_gun.dm
Comment (UnokiAs) [reply code/modules/projectiles/guns/special/syringe_gun.dm]: Totaly forgot about the bamboo ahah

**Pull request merged**
